packages/twenty-front/src/pages/settings/developers/api-keys/SettingsApiKeys.tsx

```diff
@@ -0,0 +1,74 @@
+import { SettingsPageContainer } from '@/settings/components/SettingsPageContainer';
+import { SettingsApiKeysTable } from '@/settings/developers/components/SettingsApiKeysTable';
+import { SettingsReadDocumentationButton } from '@/settings/developers/components/SettingsReadDocumentationButton';
+import { PlaygroundSetupForm } from '@/settings/playground/components/PlaygroundSetupForm';
+import { StyledSettingsApiPlaygroundCoverImage } from '@/settings/playground/components/SettingsPlaygroundCoverImage';
+import { SettingsPath } from '@/types/SettingsPath';
+import { SubMenuTopBarContainer } from '@/ui/layout/page/components/SubMenuTopBarContainer';
+import styled from '@emotion/styled';
+import { Trans, useLingui } from '@lingui/react/macro';
+import { Button, H2Title, IconPlus, MOBILE_VIEWPORT, Section } from 'twenty-ui';
+import { getSettingsPath } from '~/utils/navigation/getSettingsPath';
+
+const StyledButtonContainer = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  padding-top: ${({ theme }) => theme.spacing(2)};
+  @media (max-width: ${MOBILE_VIEWPORT}px) {
+    padding-top: ${({ theme }) => theme.spacing(5)};
+  }
+`;
+
+const StyledContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+  gap: ${({ theme }) => theme.spacing(2)};
+`;
+
+export const SettingsApiKeys = () => {
+  const { t } = useLingui();
+
+  return (
+    <SubMenuTopBarContainer
+      title={t`APIs`}
+      actionButton={<SettingsReadDocumentationButton />}
+      links={[
+        {
+          children: <Trans>Workspace</Trans>,
+          href: getSettingsPath(SettingsPath.Workspace),
+        },
+        { children: <Trans>APIs</Trans> },
+      ]}
+    >
+      <SettingsPageContainer>
+        <StyledContainer>
+          <Section>
+            <H2Title
+              title={t`Playground`}
+              description={t`Try our REST or GraphQL API playgrounds.`}
+            />
+            <StyledSettingsApiPlaygroundCoverImage />
+            <PlaygroundSetupForm />
+          </Section>
```

Comment:
There are still padding issues.

For example the gap here is 4px:
<img width="245" alt="Screenshot 2025-02-25 at 13 56 59" src="https://github.com/user-attachments/assets/71edf0e6-cfdf-4073-a50e-09399321be69" />

32px: 
![Uploading Screenshot 2025-02-25 at 13.58.45.png…]()

16px:
<img width="823" alt="Screenshot 2025-02-25 at 13 59 18" src="https://github.com/user-attachments/assets/9e612a68-a2fa-4c62-8878-3c2f5ea2494e" />

Reply:
Ah yes sorry I should have been more precise, I hadn't noticed the issue on the playground!